### ISSUE TITLE
Feature request: View bounding box coordinates

### ISSUE THREAD

**Issue by jumpinjackie** [klokantech/epsg.io#119]
What would be nice is if in addition to the coordinates (transformed or not) of the map view center, that it also shows the current bounding box in the same coordinate system

**Comment by klokan** [comment]
We have a made tool http://boundingbox.klokantech.com/ where one can draw a bounding box.
It is planned to add support for different coordinate systems...

Is this what you refer to?

Next to that, in EPSG.io, the bounding box of the coordinate system area is visible under "Projected bounds". See for example at page http://epsg.io/21781 you can see under map:
```
Projected bounds:
485071.54 75346.36
828515.78 299941.84
```
(while these are not very precisely calculated - as mentioned in bug report #85)

**Comment by jumpinjackie** [comment]
Yep, that's pretty much what I am after (but also including the missing ability to see that bbox in other coordinate systems.
